handlers/main.yml

```diff
@@ -1,6 +1,7 @@
 # handlers file for supervisor
 ---
 - name: restart supervisor
+  become: yes
```

Comment:
This does not work in Ansible < 2.

And I don't see why this could not be set the at playbook level?